### PR TITLE
update requirements.txt

### PR DESCRIPTION
**Pull request by GregorioDiStefano**
Changelog: none
Signed-off-by: Greg Di Stefano <greg.distefano+github@gmail.com>

### CODE REVIEW

**Comment by GregorioDiStefano** [comment]
@bboozzoo

File: docker-compose.client.yml [hunk 1]
```diff
@@ -6,5 +6,6 @@ services:
     #
     mender-client:
         image: mendersoftware/mender-client-qemu:master
+        privileged: true
```
Comment (bboozzoo) [inline docker-compose.client.yml]: why do we need this?

Comment (GregorioDiStefano) [inline docker-compose.client.yml]: the commit was from "testing..." :)

**Comment by GregorioDiStefano** [comment]
test

**Comment by GregorioDiStefano** [comment]
test

**Comment by GregorioDiStefano** [comment]
test

**Comment by GregorioDiStefano** [comment]
test

**Comment by bboozzoo** [comment]
Any updates to this one? If the top 2 commits are not needed, I'd suggest you dump them and we can get this merged.

**Comment by bboozzoo** [comment]
ping, updates?

**Comment by GregorioDiStefano** [comment]
no, we can close this.

**Pull request closed without merging**